examples/data-objects/webflow/src/html/formatters.ts

```diff
@@ -70,7 +68,6 @@ class HtmlFormatter extends RootFormatter<IFormatterState> {
 interface IInclusionState {
     root?: HTMLElement;
     slot?: HTMLElement;
-    view?: Promise<IFluidHTMLView>;
 }
 
 export class InclusionFormatter extends Formatter<IInclusionState> {
```

Comment:
You can probably go further and delete all the inclusion stuff entirely -- looks like no one is calling `FlowDocument.insertComponent()` which should mean that inclusions don't really exist currently.

Reply:
fixed in d9e3a6d27b08676bc8d282d4e082c45a4b31582c